samples/cpp/speech_sample/main.cpp

```diff
@@ -506,13 +506,42 @@ int main(int argc, char* argv[]) {
                             inferRequest.frameIndex = -1;
                             continue;
                         }
+                        ptrInputBlobs.clear();
+                        if (FLAGS_iname.empty()) {
+                            for (auto& input : cInputInfo) {
+                                ptrInputBlobs.push_back(inferRequest.inferRequest.get_tensor(input));
+                            }
+                        } else {
+                            std::vector<std::string> inputNameBlobs = convert_str_to_vector(FLAGS_iname);
```

Comment:
Actually I don't understand at all why we need name parameter. In my mind all input parameters should be filled with appropriate data.

@mryzhov @elilobanova what do you think?